R/release.r

```diff
@@ -40,9 +40,10 @@
 #' @param args An optional character vector of additional command
 #'   line arguments to be passed to \code{R CMD build}.
 #' @param spelling language or dictionary file to spell check documentation.
-#' See \code{\link{spell_check}}. Set to \code{NULL} to skip spell checking.
+#' See \code{\link[spelling:spell_check_package]{spelling}} package. Set to
+#' \code{NULL} to skip spell checking.
 #' @export
-release <- function(pkg = ".", check = TRUE, args = NULL, spelling = "en_US") {
+release <- function(pkg = ".", check = TRUE, args = NULL, spelling = "en_GB") {
```

Comment:
Is there a reason to change this default?

Reply:
To match the default in the `spelling` package.

Reply:
It's probably best to keep as is here, and as much as I hate to say it, it's probably to default to US english even in the spelling package (and make it an option)